app/views/vignettes/slides/_form_question.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= form.fields_for :question do |question_form| %>
+  <div>
+    <%= question_form.label :type, "Question Type" %>
+    <%= question_form.select :type, options_for_select([
+        ['No Question', ''],
+        ['Text Field', 'Vignettes::TextQuestion'],
+        ['Multiple Choice', 'Vignettes::MultipleChoiceQuestion'],
+        ['Likert Scale', 'Vignettes::LikertScaleQuestion']
+    ], question_form.object.type), {}, { id:  "question-type" } %>
+  </div>
+
+  <div id="question-text">
+    <%= question_form.label :question_text, "Question Text" %><br>
+    <%= question_form.text_area :question_text, placeholder: "Enter your question here" %>
+  </div>
+
+  <!-- The id must be the name of the corresponding class -->
+  <div id="Vignettes::TextQuestion" class="question-field" style="display: none;">
+  </div>
+
+  <div id="Vignettes::LikertScaleQuestion", class="question-field" style="display: none;">
+  </div>
+
+  <div id="Vignettes::MultipleChoiceQuestion" class="question-field" style="display: none;">
+    <div>
+      <%= question_form.label :options, "Options" %>
+      <div id="options-container">
+        <% question_form.object.options.each do |option| %>
+          <div class="option-block">
+            <%= question_form.fields_for :options, option do |option_form| %>
+              <div class="option-field">
+                <%= option_form.text_field :text, placeholder: "Option Text" %>
+                <%= option_form.hidden_field :_destroy %>
+                <button type="button" class="remove-option">Remove</button>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+        <div id="new-option-template" style="display: none;">
+          <%= question_form.fields_for :options, question_form.object.options.build, child_index: "NEW_RECORD" do |option_form|%>
+            <div class="option-field">
+              <%= option_form.text_field :text %>
+              <%= option_form.hidden_field :_destroy %>
+              <button type="button" class="remove-option">Remove</button>
+            </div>
+          <% end %>
```

Comment:
Bug: It is possible to save a slide with a multiple choice question that does not contain any option. Then, when taking that slide, the user will see `Submit Answer` even though there is nothing to choose from.